Loop/Managers/NotificationManager.swift

```diff
@@ -131,7 +131,7 @@ struct NotificationManager {
         // Give a little extra time for a loop-in-progress to complete
         let gracePeriod = TimeInterval(minutes: 0.5)
 
-        for minutes: Double in [20, 40, 60, 120] {
+        for (minutes, isCritical): (Double, Bool) in [(20, false), (40, false), (60, true), (120, true)] {
```

Comment:
[nit] Do you need the `(Double, Bool)` type or can't the compiler figure that out?

Reply:
The `Double` is needed, but could be avoided if the constants 20, 40, etc. was 20.0, 40.0, etc.